README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
+
```

Comment:
The README is a good place to include instructions for how to run your project or view it on the web. I generally delete this generated one and create one in markdown - README.md